docs/getting-started/intro-to-dockstore-tools-and-workflows.rst

```diff
@@ -1,27 +1,28 @@
 Introduction to Workflows, Tools, and Services
 ==============================================
 
+Tools and workflows hosted on Dockstore are computational steps described in languages that make it easy for their reuse in different compute environments. Generally, descriptor languages request that a user include a containerized compute environment (such as a Docker image) that packages up all of the code and dependencies needed to complete the computational step.
+
+Not all descriptor languages make a distinction between a tool and a workflow. Most users will register workflows on Dockstore. 
+
 What is a workflow?
 -------------------
 
-In general terms, a workflow is a chain of commands that has several steps linked together, which create some sort of final output. For example, a simple two-step workflow that converts a VCF into a GDS file may conceptually look like this:
+In general terms, a workflow describes a chain of commands in a specific order. For example, a simple two-step workflow that converts a VCF into a GDS file may conceptually look like this:
```

Comment:
I'm in favor of swapping out "is" for "describes," but not 
>that has several steps linked together, which create some sort of final output

with
> a chain of commands in a specific order

Strictly speaking this isn't always true. Let's say I wrote a WDL with four tasks: A, B, C, and D.
A. Change a vcf into bcf
B. Using the bcf from A to create [ASCII art of a talking cow](https://en.wikipedia.org/wiki/Cowsay) or [ASCII art of a talking Docker-associated whale](https://github.com/docker/whalesay) depending on the size of bcf file
C. Using the bcf from A to analyze overall quality 
D. Combine the output of B and C to generate a cow, or whale, telling you how good your bcf file is.

At least in WDL, when you run that in Cromwell, B and C do not have a specific order relative to each other. Both of them must go after A and before D, but if you are running on a machine that can handle concurrent jobs (ie Terra and most other things), both will execute at the same time.

Talking cows and whales are the quickest arbitrary example I can think off the top of my head but I promise there are real workflows that this sort of thing applies to.

Reply:
Maybe just ending the sentence early is the way to go?
> In general terms, a workflow describes a chain of commands.

Reply:
How about:

"In general terms, a workflow describes a set of commands that may be executed in a specified order. For example, a simple two-step workflow that converts a VCF into a GDS file may conceptually look like this:"